RELEASE_PROCESS.rst

```diff
@@ -7,9 +7,33 @@ Make the clone using:
 
 ::
 
-     git clone --branch master git@github.com:juju/juju-gui.git
-     cd juju-gui
-     git merge origin/develop
+    git clone git@github.com:juju/juju-gui.git gui-release
+    cd juju-gui
+
+Next you'll want to generate the list of changes for CHANGES.yaml.
+
+::
+
+    git log <last release>...HEAD --author 'jujugui' --format='* [%h] %b'
+
+Update CHANGES.yaml to include the list of changes--you can try to summarize
+the list of commits or just include them. If you do the latter, note that
+the list of commits may still need to be cleaned up to fit appropriately in the
```

Comment:
This wording is awkward. I think I know what you are saying, but only after reading it a few times.

```
Based on the log output, update the CHANGES.yaml file. If it makes sense, you may collapse multiple commits into a single entry in the change log.  Follow the existing formatting of the file, including bullets and spaces as the tools are non-forgiving with respect to format.